src/deluge/util/functions.cpp

```diff
@@ -501,116 +501,113 @@ char const* getPatchedParamDisplayName(int32_t p) {
 	}
 }
 
-char const* getUnpatchedParamDisplayName(int32_t p) {
+char const* getParamDisplayName(Param::Kind kind, int32_t p) {
```

Comment:
See comment below about calling this function getUnpatchedParamDisplayName

Reply:
Resolving as fix is not needed